### ISSUE TITLE
Documentation update for Mqtt Alarm Panel

### ISSUE THREAD

**Issue by tracstarr** [openhab-scripters/openhab-helper-libraries#292]
In order to use the **events.sendComman** in the example I found you have to add**from core.jsr223 import scope**. I believe this is because it's in the lib directory? Regardless, you can then call scope.events.sendCommand or use any of the other default scoped items.

**Comment by openhab-5iver** [comment]
> In order to use the events.sendComman in the example 

Could you please provide a link? I'm not sure what the MQTT Alarm Panel is.

**Comment by tracstarr** [comment]
Yes, it's [this](https://github.com/thanksmister/android-mqtt-alarm-panel) which you provide some info about [here](https://openhab-scripters.github.io/openhab-helper-libraries/Python/Community/ideAlarm/Arm%20Disarm.html#mqtt-alarm-control-panel)
